src/main/java/org/aksw/simba/squirrel/analyzer/Guess.java

```diff
@@ -0,0 +1,9 @@
+package org.aksw.simba.squirrel.analyzer;
+
+import java.io.File;
+
+
+public interface Guess {
+
+	public String guess(File data);

```

Comment:
What exactly are you guessing? The class and method should get better names to show what they are about :wink: 

Apart from that, interfaces should have javadoc comments.